Modules/Scripted/DICOMLib/DICOMWidgets.py

```diff
@@ -516,8 +516,12 @@ def promptForDatabaseDirectory(self):
       try:
         os.makedirs(databaseDirectory)
       except OSError:
-        documentsLocation = qt.QDesktopServices.DocumentsLocation
-        documents = qt.QDesktopServices.storageLocation(documentsLocation)
+        # Qt4
+        # documentsLocation = qt.QDesktopServices.DocumentsLocation
+        # documents = qt.QDesktopServices.storageLocation(documentsLocation)
+        # Qt5
```

Comment:
I'd prefer not to have the old code in comments - I don't see that it serves any purpose going forward.

Reply:
Maybe I should have marked this as `WIP:`?

Reply:
Yes, next time add a comment: "Please don't even review this yet" :)
(or just push it to your own fork?)